manifests/charts/default/templates/validatingwebhook.yaml

```diff
@@ -18,7 +18,7 @@ webhooks:
       {{- else }}
       service:
         name: istiod{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}
```

Comment:
Could you check if this works as intended ? The name doesn't include the revision - do we want the validator to point to the last istio namespace ? 

I assume the use case is to host multiple isolated istiod instances, in different namespaces ?

Reply:
@costinm I think I'd like to stay away from making _other_ changes to the chart in this particular PR, if you don't mind. From my testing in our environment, this line seems to work fine. We have tested both revisioned and non-revisioned installs, and they get the names we would expect.

Reply:
Oh for what it's worth - I think the key here is you can run multiple control planes _in the same_ namespace. This is what we are doing ... our `stable` and `canary` control planes all run in `istio-system`.